tests/test_script.py

```diff
@@ -50,3 +59,12 @@ def test_min_confidence():
     assert call_vulture([
         'vulture/core.py', '--exclude', 'whitelists',
         '--min-confidence', '100']) == 0
+
+
+def test_exclude():
+    expected_output = """\
+vulture/lines.py:40: unused function 'get_last_line_number' (60% confidence)
```

Comment:
Checking the output is brittle. Whenever we add or remove a line before line 40, we need to adapt this test.

Can we just check the exit code instead?

`['vulture/', '--exclude', 'vulture/core.py,vulture/utils.py']` -> 1
`['vulture/', '--exclude', 'vulture/core.py,vulture/utils.py,vulture/lines.py']` -> 0